WooCommerce/Classes/ViewRelated/Dashboard/Settings/SettingsViewController.swift

```diff
@@ -139,32 +139,30 @@ private extension SettingsViewController {
         let storeRows: [Row] = sites.count > 1 ?
             [.selectedStore, .switchStore] : [.selectedStore]
 
-        let otherSection: Section
-        #if DEBUG
-        otherSection = Section(title: otherTitle, rows: [.appSettings, .wormholy], footerHeight: CGFloat.leastNonzeroMagnitude)
-        #else
-        otherSection = Section(title: otherTitle, rows: [.appSettings], footerHeight: CGFloat.leastNonzeroMagnitude)
-        #endif
+        sections = [
+            Section(title: selectedStoreTitle, rows: storeRows, footerHeight: CGFloat.leastNonzeroMagnitude),
+            Section(title: nil, rows: [.support], footerHeight: UITableView.automaticDimension),
```

Comment:
The distance between the cells for "Help and Support" and "manage Card Readers" seems too high, specially compared to the distance between "Switch Store" and "Help And Support". 

<img src="https://user-images.githubusercontent.com/2722505/107432293-be780300-6af5-11eb-8db5-9b3a4f4dfd4e.png" width="350"/>

The footerHeight for the support section might need to be `CGFloat.leastNonzeroMagnitude` as well when we present the "Manage card readers" row

Reply:
Fixed in https://github.com/woocommerce/woocommerce-ios/pull/3624/commits/466de0b9d7cf649217ebc0382809c755f5e62528